tools/serviced-storage/driver.go

```diff
@@ -200,11 +200,15 @@ func (c *CheckOrphans) Execute(args []string) error {
 
 	//  we retrieve the list of all devices that CC has access to
 	var ccDevices []string
-	for _, v := range drv.List() {
-		vol, err := drv.GetVolume(v, false)
+	for _, v := range drv.ListTenants() {
+		_vol, err := drv.Get(v)
 		if err != nil {
 			return err
 		}
+		vol, ok := _vol.(*devicemapper.DeviceMapperVolume)
+		if !ok {
+			log.Fatal("Volume", v, "could not be cast as devicemapper.DeviceMapperVolume")
```

Comment:
This is user facing, so change this to something like, "volume is not of DeviceMapper type" or something